### DISCUSSION TITLE
Use GitHub for discussions instead of Slack ?

### DISCUSSION THREAD

**Discussion by JBBianchi** [serverlessworkflow/specification#871] [General]
Hello,

Nowadays, discussions tend to happen on Slack. It's fine but lacks a bit of visibility and accessibility. If, in a year, a new contributor asks about topic X, it's gonna be difficult to find the related discussion. I suggest relying more on GitHub discussion for archiving purpopses.

**Comment by cdavernas** [comment]
I totally agree! @fjtirado @ricardozanini Let's try to bring our conversations in GH conversations, instead of issues, PRs and slack ;)

**Comment by fjtirado** [comment]
Ok, I opened discussion https://github.com/serverlessworkflow/specification/discussions/872 for do:

**Comment by cdavernas** [comment]
Closing the present conversation, as its content will be covered by the PR related to https://github.com/serverlessworkflow/specification/discussions/887
